packages/dev/core/src/Misc/tools.ts

```diff
@@ -417,9 +450,57 @@ export class Tools {
     }
 
     /**
-     * Load a script (identified by an url). When the url returns, the
+     * Get a script URL including preprocessing
+     * @param scriptUrl the script Url to process
+     * @returns a modified URL to use
+     */
+    public static GetScriptUrl(scriptUrl: Nullable<string>, forceAbsoluteUrl?: boolean): string {

```

Comment:
Maybe GetBabylonScriptURL ? to prevent confusion ?

Reply:
Sure, on it

Reply:
Done